cvat/apps/iam/permissions.py

```diff
@@ -645,6 +645,8 @@ def get_scopes(request, view, obj):
             ('annotations', 'PATCH'): 'update:annotations',
             ('annotations', 'DELETE'): 'delete:annotations',
             ('annotations', 'PUT'): 'update:annotations',
+            ('annotations', 'POST'): 'import:annotations',
+            ('append_annotations_chunk', 'PATCH'): 'update:annotations',
```

Comment:
I suppose we need to add HEAD method here:
```('append_annotations_chunk', 'HEAD'): 'update:annotations'```
It's also a part of TUS protocol, even though it does not generate an error in our case